src/base/components/DeviceProfile.js

```diff
@@ -26,7 +28,9 @@ class DeviveProfileTab extends React.Component {
               <div>Height: {productData? productData.height: ''}</div>
               <div>Weight: {productData? productData.weight: ''}</div>
               <div>width: {productData? productData.width: ''}</div>
-            </div>: null
+            </div>: <img src={productData.image ? productData.image: 'http://bit.do/not-found'} width="640" height="350"/>
```

Comment:
Why you do not read carefully? Use
https://default.wrioos.com/img/no-photo-200x200.png